clients/java/src/main/java/io/camunda/zeebe/client/impl/search/response/OperationImpl.java

```diff
@@ -30,8 +30,8 @@ public class OperationImpl implements Operation {
   public OperationImpl(final OperationItem item) {
     id = item.getId();
     batchOperationId = item.getBatchOperationId();
-    type = item.getType().toString();
-    state = item.getState().toString();
+    type = (item.getType() == null) ? null : item.getType().toString();
```

Comment:
We should use enums to be consistent with the API spec

Reply:
I did not add enums anymore, because in the model we return strings, so there is not need to convert from enum to enum, just from enum to string.